WireUI/Sources/WireSidebar/Views/Preview/SidebarPreview.swift

```diff
@@ -0,0 +1,74 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+import WireFoundation
+
+@available(iOS 17.0, *)
```

Comment:
why iOS 17 is there an alternate component for previous version?

Reply:
so that's only for previews right?

Reply:
Yes, this is only for previewing the view. 
The preview uses `NavigationSplitView` which is available from iOS 17 only.
In the project we currently use the `UISplitViewController`.